Core/Loaders.cpp

```diff
@@ -355,3 +355,41 @@ bool LoadFile(FileLoader **fileLoaderPtr, std::string *error_string) {
 	coreState = CORE_ERROR;
 	return false;
 }
+
+bool UmdReplace(std::string filepath, char *error) {
+	IFileSystem* currentUMD = pspFileSystem.GetSystem("disc0:");
+
+	if (!currentUMD) {
+		error = "has no disc";
+		return false;
+	}
+
+	FileLoader *loadedFile = ConstructFileLoader(filepath);
+
+	if (!loadedFile->Exists()) {
+		delete loadedFile;
+		sprintf(error, "%s doesn't exist", loadedFile->Path().c_str());
```

Comment:
`error` isn't allocated with any memory (it starts as an empty string), so we can't do this.  I'd recommend making it `std::string &` or `std::string *`.

-[Unknown]